js/app.js

```diff
@@ -1,4 +1,6 @@
-var ingredients = {
+/*global m, L*/
```

Comment:
What does this comment mean?

Reply:
These are the libraries we use: mithril and Leaflet. This is required for eslint, as it doesn't allow global variables otherwise

I added a comment to make this more clear